cellprofiler/gui/pipelinelistview.py

```diff
@@ -1117,7 +1117,9 @@ def __init__(self, *args, **kwargs):
         # Gap before and after text
         self.text_gap = 3
         # The height of one row in the display
-        self.line_height = 16 + self.border + self.gap
+        self.line_height = max(
```

Comment:
The font changed and that made each line a bit too small, cutting off descenders. This bases the height on the height of the text or the icons, whichever is larger.